distributed/nanny.py

```diff
@@ -384,7 +384,7 @@ async def instantiate(self, comm=None) -> Status:
                 raise
         return result
 
-    async def restart(self, comm=None, timeout=2, executor_wait=True):
+    async def restart(self, comm=None, timeout=20, executor_wait=True):
```

Comment:
While I doubt this will be critical, this changes the behaviour for users. I would expect restart to not be called very frequently in CI. Any reason to increase this in the code itself and not the test?

Reply:
This timeout is actually ignored by users, as in a typical non-test situation you get

distributed.comms.timeouts.connect (was 10s; now raised to 20s)
-> Client.restart without arguments
-> Scheduler.restart
-> multiply by 0.8 in Scheduler.restart before passing it down to Nanny
-> Nanny.restart